test/integration.toml

```diff
@@ -24,7 +24,9 @@ hash_type = "Data"
 # the directory containing this config file.
 [genesis.system_cells]
 files = [
-  { file = "cells/always_success" }
```

Comment:
Each integration spec can modify chainspec, maybe we don't need to include the two cells in every specs.

Reply:
Most of the test-cases don't care about the system cells.
I think that Including these two system cells is OK for integration-tests.